.github/ISSUE_TEMPLATE/BugReport.yml

```diff
@@ -0,0 +1,94 @@
+name: Bug Report
+description: Something's not working right in Charmed Kafka? Use this 
+  template.
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for pointing out this issue. Please provide info below so we could diagnose and fix it fast.
+
+  - type: textarea
+    id: Description
+    attributes:
+      label: Description
+      description: "Please describe the bug for us: "
+      placeholder: ex. "charmed kafka is breaking with this field set to value."
+
+  - type: dropdown
+    id: Urgency
+    attributes:
+      label: Urgency
+      description: How urgent is this issue for you?
+      options:
+        - Casually reporting
+        - Annoying bug in our test suite
+        - Blocker for our release
+        - Impacting a customer, please help now!
+    validations:
+      required: true
+
+  - type: input
+    id: Juju-CLI-Version
+    attributes:
+      label: Juju client version
+      description: Version of the Juju client. Run `juju version` to obtain it.
+      placeholder: 3.1.7
+    validations:
+      required: true
+
+  - type: input
+    id: Juju-Version
+    attributes:
+      label: Juju client version
```

Comment:
Juju controller version?

Reply:
Sorry. I messed up a bit the template on this item and the next one